internal/provider/resource_docker_service.go

```diff
@@ -944,7 +944,7 @@ func resourceDockerServiceV0() *schema.Resource {
 		// callbacks are no longer relevant
 		Schema: map[string]*schema.Schema{
 			"auth": {
-				Type:     schema.TypeMap,
+				Type:     schema.TypeList,
```

Comment:
In my understanding, we should not change the old schema.

Reply:
Oh, it was solved.

Reply:
Yes, I reverted the change :)